### DISCUSSION TITLE
Error Storage backend has reached its minimum free disk threshold.

### DISCUSSION THREAD

**Discussion by jolumape** [minio/minio#15133] [Q&A] [answered]
Hi Dears,

MinIO has been installed on a Linux server assigning it a 3 TB unit, of which only 1.4 TB has been used, however for certain cases of file upload through the API, it returns the following error:

Storage backend has reached its minimum free disk threshold. Please delete a few objects to proceed.

The version of MinIO:

minio version RELEASE.2022-06-20T23-13-45Z (commit-id=b3ebc69034bff982ca2c1a6ca43afd20838e99d5)
Runtime: go1.18.3 linux/amd64
Copyright (c) 2015-2022 MinIO, Inc.
Licence AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>

The MinIO configuration is as follows:

MINIO_VOLUMES="/repo_minio/minio/"
MINIO_OPTS="--address 192.168.1.22:9000 --console-address 192.168.1.22:9090"

Mapped drive status for the MinIO repository:

Filesystem               		            Size     Used     Avail     Use%     Mounted on
/dev/mapper/vgdatos-lvrepominio      3.0T     1.4T     1.5T       50%       /repo_minio

It is also observed that the 2 created buckets are shown in the Web console, but it incorrectly shows the total amount of uploaded files and the total amount of disk used.

![image](https://user-images.githubusercontent.com/75599490/174827785-bf58538b-c1d9-4aec-8123-7b58b94a91aa.png)

I would be grateful if you could help me in solving this problem.

Thanks 

Jose

**Comment by harshavardhana** [comment, marked as answer]
> /dev/mapper/vgdatos-lvrepominio 3.0T 1.4T 1.5T 50% /repo_minio

Is this ext4?

**Reply by jolumape** [reply]
Dear Harshavardhana

Indeed the unit /dev/mapper/vgdatos-lvrepominio is ext4

![image](https://user-images.githubusercontent.com/75599490/174911330-6b55a6fe-2c15-4c6c-a290-591bd00ef92c.png)

Is this an inconvenience?

Thanks for your support.

Jose

**Reply by harshavardhana** [reply]
Yes you should use our recommend FS - xfs - you are hitting ext4 directory limits - https://blog.merovius.de/posts/2013-10-20-ext4-mysterious-no-space-left-on/

**Reply by jolumape** [reply]
Harshavardhana thanks for your answer.

Understood, we will proceed to create a new drive with XFS.

I would ask for your support with some additional queries:

When migrating the information to the new drive, it is also necessary to migrate the ".minio.sys" directory.

Would this change also solve the problem of not displaying correctly the number of objects and the total disk used in the Web console? Or is there any way to regenerate this information?

![image](https://user-images.githubusercontent.com/75599490/175090257-43427a05-97a5-4114-93df-7eebfd8b90c8.png)

Thanks

Jose

**Reply by jolumape** [reply]
Dear @harshavardhana 

In the next few days we will be doing the assignment of a unit with fs xfs as you recommended.

Also tell you that a test was made to create a new bucket and try to load there the files that previously gave the error **Storage backend has reached its minimum free disk threshold**.

In this case there were no problems in loading. 

As I told you in my first communication, this problem occurs only with some files.

For reference, on average are uploaded: 

271114 files are per day (20 hours of active service)
13556 files per hour
226 files per minute.

Could this be part of the cause of the problem?

Likewise, I would appreciate it if you could tell me how I could regenerate the information on the number of objects and the space used, since incorrect information is being displayed on the Web console.

![image](https://user-images.githubusercontent.com/75599490/175425155-b3025010-41ab-4ab2-850a-0396058d0120.png)

Thanks

Jose

**Comment by 5hanth** [comment]
I get this error while trying to upload files to a docker volume that is bind to a path in host which in turn is a mount point for a Digital ocean spaces using s3fs.

**Reply by harshavardhana** [reply]
you shouldn't be running MinIO on top of s3fs.

**Comment by klauspost** [comment]
We were able to track down an issue that could lead to this. Upgrading to latest is recommended.
